pynestml/grammars/PyNestMLParser.g4

```diff
@@ -100,12 +100,16 @@ parser grammar PyNestMLParser;
 
   logicalOperator : (logicalAnd=AND_KEYWORD | logicalOr=OR_KEYWORD );
 
+  indexParameter : (sizeStr=NAME | sizeInt=UNSIGNED_INTEGER);
```

Comment:
Should we check somewhere whether the size >0 in the declaration? This brings up the question whether vectors should be resizable. Probably not, but it might be good to discuss what this would entail in any case.